sky/data/data_utils.py

```diff
@@ -521,17 +521,32 @@ def get_gsutil_command() -> Tuple[str, str]:
 def run_upload_cli(command: str, access_denied_message: str, bucket_name: str):
     # TODO(zhwu): Use log_lib.run_with_log() and redirect the output
     # to a log file.
```

Comment:
To keep things simple, is it possible to just use log_lib.run_with_log?

Reply:
I didn't do that at first because I found that this function would be triggered by a multiprocessing pool, which could potentially corrupt the log file. However, since the current work directory sync also uses multiprocessing with rsync log to the same file, I assume it's not a problem for now. Let's leave it as it is and use `log_lib.run_with_log()`